Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -654,6 +654,11 @@ private void InternalUpdate()
                 if (rollOffTimer >= rollOffTime)
                 {
                     HasPress = false;
+
+                    if (HasGesture)
```

Comment:
@julenka @thalbern do you guys know if HasGestureMax is also something that we'd want to consider resetting here? The naming here is so similar that I can't help but ask.

Reply:
I agree that this should be reset too , but having a look at the code (get/set for property, deprecated set method, setting flag directly) I don't see anywhere in MRTK where we would actually set that flag. Is this wired up in the inspector somehow?

Reply:
@wiwei well HasGestureMax is a bit problematic and separate bug. HasGestureMax has comment "Currently not controlled by Interactable directly" but we do reset it in one method. So it should be either or. 

@vaoliva suggestion, just make it HasPress = false, and HasGesture = false. It's weird to check one but not the other and there are only two values for it so setting it to false is fine. We also don't save on performance or anything. 

@julenka / @thalbern , I wonder if we can remove the roll off timer? The comments suggest we are delaying firing some lost focus but really we are just delaying setting HasPress to false when we are pressed but lose focus which I currently don't understand why we have to delay that setting. It's weird to me that we can have HasFocus = false and HasPress = true for 0.25 seconds

```
        // allows for switching colliders without firing a lose focus immediately
        // for advanced controls like drop-downs
        protected float rollOffTime = 0.25f;
        protected float rollOffTimer = 0.25f;
```

Reply:
@Troy-Ferrell wouldn't it be better to call ResetInputTrackingStates() at that spot to make sure all the interaction states are reset?

Reply:
Yea I considered that. It makes sense but it's just a larger change and not sure of the impact.